src/main/java/net/dv8tion/jda/api/utils/FileUpload.java

```diff
@@ -358,6 +364,70 @@ public FileUpload setDescription(@Nullable String description)
         return this;
     }
 
+    /**
+     * Turns this attachment into a voice message with the provided waveform.
+     *
+     * @param  mediaType
+     *         The audio type for the attached audio file. Should be {@code audio/ogg} or similar.
+     * @param  waveform
+     *         The waveform of the audio, which is a low frequency sampling up to 256 bytes.
```

Comment:
Would it be a good idea to add a bit of sample code to get this waveform from the file?